pkg/verifier/notaryv2/notaryv2.go

```diff
@@ -73,9 +77,8 @@ func (f *notaryv2VerifierFactory) Create(version string, verifierConfig config.V
 		return nil, fmt.Errorf("failed to parse config for the input: %v", err)
 	}
 
```

Comment:
small nit: but can we delete the comment at Line 75. Looks like it's it was accidentally left over :)